### PR TITLE
LRQA-82564

### PR DESCRIPTION
**Pull request by crislucas1**
https://liferay.atlassian.net/browse/LRQA-82564

### CODE REVIEW

**Comment by crislucas1** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+	}
+
+	@description = "Check if it is possible to delete a blog"
+	@priority = 4
+	test CanDeleteBlog {
+		task ("Go to the blogs page") {
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Guest");
+		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase:80]: Hey @crislucas1, I think this step is already in your `setUp`, I think I might not need to repeat it here again. What do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+				entryContent = "Blogs Entry Content",
+				entryTitle = "Blogs Entry Title");
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blogs Entry2 Content",
+				entryTitle = "Blogs Entry2 Title");
+
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Guest");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase:109]: Hey @crislucas1, when you validate that the blogs are present on the page, you are already accessing the Blogs Page page at that moment. I don't think it's necessary to put this navigation here again. What do you think?

**Review by ViniLopes20 — CHANGES_REQUESTED** [review]
Hey @crislucas1, could you check my comments above? Remember to always review your test to check if all the macros/steps you added make sense or not. Sometimes we add a repeated step that does not affect the test result, but it may not make sense to repeat it and make the code longer

**Comment by crislucas1** [comment]
Right, thanks @ViniLopes20 . I'll correct

**Comment by crislucas1** [comment]
ci:test:sf

**Pull request closed without merging**